app/org/sagebionetworks/bridge/stormpath/StormpathAccount.java

```diff
@@ -201,6 +229,7 @@ private void encryptJSONTo(String key, Object value) {
     private void encryptTo(String key, String value) {
         if (value == null) {
             acct.getCustomData().remove(key);
+            acct.getCustomData().remove(key+VERSION_SUFFIX);
```

Comment:
Is there anything we can add to the object model so we can verify in our tests we're clearing the key+VERSION_SUFFIX attribute?